ttnn/cpp/ttnn/operations/moreh/moreh_layer_norm/moreh_layer_norm.cpp

```diff
@@ -0,0 +1,33 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "moreh_layer_norm.hpp"
+
+#include "ttnn/operations/moreh/moreh_layer_norm/device/moreh_layer_norm_device_operation.hpp"
+
+namespace ttnn::operations::moreh::moreh_layer_norm {
+std::vector<std::optional<Tensor>> MorehLayerNorm::invoke(
+    const Tensor& input,
+    const uint32_t normalized_dims,
+    const float eps,
+    const std::optional<const Tensor> gamma,
```

Comment:
Use `const std::optional<const Tensor>&` for consistency in this ops.
Optional: `const std::optional<Tensor>&` to align with other ops.

Reply:
Resolved